source/topics/troubleshooting/support-ticket.md

```diff
@@ -0,0 +1,11 @@
+When submitting a support request to Aptible, keep the following best practices in mind:
+
+1) If you provide us with as much detail as possible we'll be able to troubleshoot your issue much faster.  When appropriate, please provide  
+
+- Relevant handles (app, account, database etc) 
+- Logs or error messages 
+- The date(s)/time(s) (UTC) when you experienced a problem
+
+2) Sanitize any sensitive information. This includes database connection URLs, SSH keys, passwords, tokens, and any secure environment variables you may be using. 
+
+3) We love getting formatted support requests because we're able to parse important information much easier.  Using backticks for monospacing or codeblocks is a great way to make information clear. You can either send us a ticket through the Zendesk webportal or include a [gist](https://gist.github.com/) in your request. 
```

Comment:
Looks like md actually isn't supported through the [portal](https://aptible.zendesk.com/hc/en-us/requests/new) either.  Maybe recommend gist?